src/PropTree.ts

```diff
@@ -36,40 +33,35 @@ export class PropTree
         }
 
         // build the tree of props
-        for (let i = 0; i < props.length; ++i)
-        {
+        for (let i = 0; i < props.length; ++i) {
             const prop = props[i];
 
-            if (!prop || !prop.name)
-                continue;
+            if (!prop || !prop.name) continue;
```

Comment:
I'd rather keep these as separate lines.

Reply:
In order to keep these as separate lines with prettier would need to add in braces. 

Prettier [wont do that out of principle](https://prettier.io/docs/en/rationale.html#what-prettier-is-not-concerned-about) but there's an [eslint rule](https://eslint.org/docs/rules/curly) that can auto-add this.